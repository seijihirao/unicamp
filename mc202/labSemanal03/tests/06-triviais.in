i 1 0
v 0 0
p
i 11 1
v 0 1
p
i 11 2
v 0 2
p
i 11 3
v 0 3
p
i 11 4
v 0 4
p
v 0 0
v 1 1
v 2 2
v 3 3
v 4 4
p
p
t
