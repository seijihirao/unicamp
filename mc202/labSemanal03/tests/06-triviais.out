0.0000 
1.0000 0.0000 
2.0000 0.0000 1.0000 
3.0000 1.0000 0.0000 2.0000 
4.0000 2.0000 0.0000 1.0000 3.0000 
4.0000 2.0000 0.0000 1.0000 3.0000 
4.0000 2.0000 0.0000 1.0000 3.0000 
